### ISSUE TITLE
[Docs] Move Loki Operator documentation into /docs/sources/

### ISSUE THREAD

**Issue by JStickler** [grafana/loki#8812]
Move the documentation for the Loki Operator from **/operator/docs** to **/docs/sources/setup/install**
This task is dependent on #8712 being completed first.  [COMPLETED]

Currently published independently -> https://loki-operator.dev/docs/

**Comment by JStickler** [comment]
Closing as Won't Fix
